cdap-ui/app/cdap/components/Experiments/store/ExperimentsListActionCreator.js

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import experimentsStore, {ACTIONS, MMDS_SORT_METHODS} from 'components/Experiments/store';
+import {setAlgorithmsList} from 'components/Experiments/store/SharedActionCreator';
+import {myExperimentsApi} from 'api/experiments';
+import {getCurrentNamespace} from 'services/NamespaceStore';
+
+function setExperimentsLoading() {
+  experimentsStore.dispatch({
+    type: ACTIONS.SET_EXPERIMENTS_LOADING
+  });
+}
+
+function setExperimentsListError(error = null) {
+  experimentsStore.dispatch({
+    type: ACTIONS.SET_ERROR,
+    payload: {
+      error
+    }
+  });
+}
+
+function getExperimentsList() {
+  setExperimentsLoading();
+  let {
+    offset,
+    limit,
+    sortMethod,
+    sortColumn
+  } = experimentsStore.getState().experiments;
+  myExperimentsApi
+    .list({
+      namespace: getCurrentNamespace(),
+      offset,
+      limit,
+      sort: `${sortColumn} ${sortMethod}`
+    })
+    .subscribe(res => {
+      let experiments = res.experiments;
```

Comment:
nit: This can be,
```
let {experiments, totalCount: totalRowCount} = res;
```

Reply:
Fixed, but i think it should be `totalRowCount: totalCount`. I get a lint error if it's the other way around.